x-pack/legacy/plugins/siem/public/containers/case/configure/use_configure.test.tsx

```diff
@@ -23,113 +28,85 @@ describe('useConfigure', () => {
     jest.restoreAllMocks();
   });
 
-  const args = {
-    setConnector: jest.fn(),
-    setClosureType: jest.fn(),
-    setCurrentConfiguration: jest.fn(),
-  };
-
   test('init', async () => {
     await act(async () => {
       const { result, waitForNextUpdate } = renderHook<string, ReturnUseCaseConfigure>(() =>
-        useCaseConfigure(args)
+        useCaseConfigure()
       );
       await waitForNextUpdate();
       expect(result.current).toEqual({
-        loading: true,
-        persistLoading: false,
+        ...initialState,
         refetchCaseConfigure: result.current.refetchCaseConfigure,
         persistCaseConfigure: result.current.persistCaseConfigure,
+        setCurrentConfiguration: result.current.setCurrentConfiguration,
+        setConnector: result.current.setConnector,
+        setClosureType: result.current.setClosureType,
+        setMapping: result.current.setMapping,
       });
     });
   });
 
   test('fetch case configuration', async () => {
     await act(async () => {
       const { result, waitForNextUpdate } = renderHook<string, ReturnUseCaseConfigure>(() =>
-        useCaseConfigure(args)
+        useCaseConfigure()
       );
       await waitForNextUpdate();
       await waitForNextUpdate();
       expect(result.current).toEqual({
+        ...initialState,
+        closureType: caseConfigurationCamelCaseResponseMock.closureType,
+        connectorId: caseConfigurationCamelCaseResponseMock.connectorId,
+        connectorName: caseConfigurationCamelCaseResponseMock.connectorName,
+        currentConfiguration: {
+          closureType: caseConfigurationCamelCaseResponseMock.closureType,
+          connectorId: caseConfigurationCamelCaseResponseMock.connectorId,
+          connectorName: caseConfigurationCamelCaseResponseMock.connectorName,
+        },
+        version: caseConfigurationCamelCaseResponseMock.version,
+        firstLoad: true,
         loading: false,
-        persistLoading: false,
         refetchCaseConfigure: result.current.refetchCaseConfigure,
         persistCaseConfigure: result.current.persistCaseConfigure,
+        setCurrentConfiguration: result.current.setCurrentConfiguration,
+        setConnector: result.current.setConnector,
+        setClosureType: result.current.setClosureType,
+        setMapping: result.current.setMapping,
       });
     });
   });
 
-  test('fetch case configuration - setConnector', async () => {
```

Comment:
Why those tests are not needed?

Reply:
setConnector is no longer mocked. by checking the updates to state, we're checking that the dispatch functions were called correctly